docs/changelog.md

```diff
@@ -1,5 +1,73 @@
 ## Changelog
 
+### v2.1.0 (8/6/2020)
+
+**Highlights**
+- Support new backbones: [RegNetX](https://arxiv.org/abs/2003.13678), [Res2Net](https://arxiv.org/abs/1904.01169)
+- Support new methods: [NASFCOS](https://arxiv.org/abs/1906.04423), [PISA](https://arxiv.org/abs/1904.04821)
+- Support new dataset: [LVIS](https://arxiv.org/abs/1908.03195)
+
+**Bug Fixes**
+- Fix docstring styles (#2634)
+- Fix the bug of model zoo url in README.md (#2641)
+- Fix validate default (#2651)
```

Comment:
The description is unclear.
Change `--validate` to `--no-validate` to enable validation after training epochs by default.